ui/src/containers/configs/NewConfig.js

```diff
@@ -62,6 +63,22 @@ class NewConfig extends Component {
         connectionValue: "",
       },
       configCreated: false,
+      showPayloadEditor: false,
+      payloadEditorChanges: false,
+      editorConfig: {
```

Comment:
This seems to be a duplicate from `state.config`. Do we really need it two times?

Reply:
Good catch, as we have to treat the editor content as a string only. I replaced it with an empty string. We `JSON.stringify` upon switching to the editor.